static/src/stylesheets/module/content-garnett/_article-immersive.scss

```diff
@@ -645,14 +647,37 @@
     .content__headline--immersive {
         @include fs-headline(7, true);
         line-height: 1.1;
-        color: #ffffff;
+        color: $garnett-neutral-1;
         padding: 0;
         padding-bottom: $gs-baseline * 2;
         font-weight: 700;
 
         @include mq(desktop) {
             font-size: 50px;
         }
+
+        &:before {
+            position: absolute;
```

Comment:
Can we add a comment explaining what this is? Its just nice when we are reading the code in the feature :)

Reply:
done.